ceph_deploy/tests/util.py

```diff
@@ -24,5 +26,12 @@ class Empty(object):
     instantiation.
     """
     def __init__(self, **kw):
-        for k, v in kw.items():
+        for k, v in list(kw.items()):
             setattr(self, k, v)
+
+
+def assert_too_few_arguments(err):
```

Comment:
I didn't want to repeat the clunky double check. It appears in many places in the code base. So I just put this piece of code in a function.

Now, the reason that this new check is needed in the first place  is simply that *argparse* now has different error messages, and the tests failed expecting the ones from Python 2.x.